### PR TITLE
do not fail uptime check for kibana or redis

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

File: seqr/views/status.py [hunk 1]
```diff
@@ -26,7 +27,7 @@ def status_view(request):
     try:
         redis.StrictRedis(host=REDIS_SERVICE_HOSTNAME, socket_connect_timeout=3).ping()
     except Exception as e:
-        dependent_services_ok = False
+        secondary_services_ok = False
```
Comment (ShifaSZ) [inline seqr/views/status.py:30]: Is Redis also a secondary service? It looks like a dependent one.

File: seqr/views/status.py [hunk 2]
```diff
@@ -43,11 +44,11 @@ def status_view(request):
         if resp.status >= 400:
             raise ValueError('Error {}: {}'.format(resp.status, resp.reason))
     except Exception as e:
-        dependent_services_ok = False
+        secondary_services_ok = False
         logger.error('Kibana connection error: {}'.format(str(e)))
```
Comment (ShifaSZ) [inline seqr/views/status.py:48]: Don't you think it is better to change the log from an `error` to a `warning`?

**Review by ShifaSZ — CHANGES_REQUESTED** [review]

File: seqr/views/status.py (hunk 2 above)
Comment (hanars) [reply seqr/views/status.py:48]: no I need to be alerted if these services are down, its really bad if it happens

File: seqr/views/status.py (hunk 1 above)
Comment (hanars) [reply seqr/views/status.py:30]: seqr runs fine if redis is down, just a bit slower

**Review by ShifaSZ — APPROVED** [review]

**Pull request merged**